components/CompanyDetailsCard/CompanyDetailsCard.tsx

```diff
@@ -19,17 +24,89 @@ type CompanyDetailsCardProps = {
   company: Company;
 };
 
+function extractDomain(url: string) {
+  const parts = url?.split("www.");
+  if (parts?.length >= 2) {
+    const domain = parts[1];
+    return domain.replace(/\/$/, ""); // Remove trailing "/"
+  } else {
+    return null; // "www." not found in the URL
+  }
+}
+
 const CompanyDetailsCard: React.FC<CompanyDetailsCardProps> = ({ company }) => {
+  const [displayWebsite, setDisplayWebsite] = useState(false);
   return (
     <div className={styles.detailsCardMainContainer}>
-      <h2>{company?.name}</h2>
-      <h4>City: {company?.city}</h4> <h4>Activity: {company?.category}</h4>
-      <h4> {company?.website}</h4>
-      <h5>IND number: {company?.indReferentNumber}</h5>
-      <h5>
-        Address:{" "}
-        {`${company?.street}, ${company?.houseNumber}. ${company?.city}. ${company?.postCode}.`}
-      </h5>
+      <div className={styles.topBar}>
```

Comment:
This navbar looks different than the one on homepage. 
Maybe we should make it reusable and use the same one everywhere?

![Screenshot 2023-09-04 at 10 38 18](https://github.com/luizfiorentino/fndr/assets/20372832/dc0bc673-309e-481b-bf1a-e3eab39b54c9)
![Screenshot 2023-09-04 at 10 38 28](https://github.com/luizfiorentino/fndr/assets/20372832/f56343de-56ed-4ee6-a6ad-a5712743144e)

The one you created is closer to the styling in the design, but it doesn't use the actual avatar of the user ...